app/bundles/FormBundle/Entity/SubmissionRepository.php

```diff
@@ -190,9 +191,10 @@ public function getEntity($id = 0)
                 ->from($this->getResultsTableName($form->getId(), $form->getAlias()), 'r')
                 ->where('r.submission_id = :id')
                 ->setParameter('id', $id);
-            $results = $q->execute()->fetchAll();
```

Comment:
@escopecz please confirm that this change will not change the functionality

Reply:
It looks correct to me.